### PR TITLE
initial birdnet model result analysis

### PR DESCRIPTION
**Pull request by Siying6560**
1. update the readme file

2. create a notebook folder: 

- copy Anthony "2022-10-30-birdnet-analyze-v1" notebook
- create a notebook named "birdnet_result_analyze" and run some preliminary result analysis

### CODE REVIEW

File: users/siyingliu/README.md [hunk 1]
```diff
@@ -1 +1,11 @@
 # Siying README file
+
+1. load up Birdnet model result in parquet file
+2. run initial analysis
+
+- bird species anlaysis:
```
Comment (acmiyaguchi) [inline users/siyingliu/README.md]:
```suggestion
- bird species analysis:
```

**Review by acmiyaguchi — APPROVED** [review]
Overall looks good to me. Just a few nits for consistency:

- `notebooks` instead of `notebook` for the directory (minor nit)
- If you don't modify the notebook, I don't think it's necessary to copy it over, and you can instead reference a link to it in your other notebook. If you do copy it over, it'd be nice to include attribution.
- Putting a timestamp on the notebook can be useful for organizing, if you end up having several notebooks. If you plan on one big notebook, it's fine as-is.

Regarding analysis:

- Nice job on the high level exploration, I learned some new things here. 
  - The power law distribution of bird species listed in the dataset is (in hindsight) unsurprising, since it follows a similar distribution to the length of audio per species.
  - I liked the confidence vs species count scatter plot, it's a neat presentation of the data.

Here are two things that I'm curious about:

- What are confidences on a per species basis? I imagine limiting the same analysis on a single species of bird will show which birds the model gets it mixed up with (or other birds in the audio that happen to be colocated).
- What percentage of each audio track contains birdcalls vs noise. This one needs the timing of each audio track, which might need to be created by looping over all of the audio tracks.

File: users/siyingliu/README.md (hunk 1 above)
Comment (Siying6560) [reply users/siyingliu/README.md]: update and git push it. don't know whether I need to create a new PR request?

Comment (acmiyaguchi) [reply users/siyingliu/README.md]: No you don't need a new PR, you can just stack the new commits on top of this one.

**Comment by acmiyaguchi** [comment]
Looks good to me. Feel free to open up new PRs if there's anything you want to change.

**Pull request merged**